routes/v1.js

```diff
@@ -87,6 +88,7 @@ function addRoutes(app, peliasConfig) {
       postProc.renamePlacenames(),
       postProc.parseBoundingBox(),
       postProc.normalizeParentIds(),
+      postProc.assignLabels(require('pelias-labels')),
```

Comment:
remove the require from here since we now have the default inside the labels module.